many false positives in AWS018 non-empty description

Hi! 

In a lot of description fields, I am joining strings, e.g. 
`description = join(" ", [var.application_name, "ec2", "egress HTTP"])`

This yields to many false positive since the newest version
(I guess https://github.com/tfsec/tfsec/commit/5ac30379899ab159ded2ec3141ad49381b41a1e3#diff-82e1fc6c5b3ecfa3083f4f2c9532e1a33c03c516630474130ed0f40b31451029)

`[AWS018][ERROR] Resource 'aws_security_group_rule.sg_ec2_outbound_http' should include a non-empty description for auditing purposes.`

**Expected behavior**
AWS0018 not throwing an error.

**Additional context**
Using macOS, with tfsec installed using brew, tfsec@0.39.6

--- a/internal/app/tfsec/parser/attribute.go
+++ b/internal/app/tfsec/parser/attribute.go
@@ -235,6 +235,8 @@ func (attr *Attribute) IsEmpty() bool {
 	}
 	if attr.Value().IsNull() {
 		switch t := attr.hclAttribute.Expr.(type) {
+		case *hclsyntax.FunctionCallExpr:
+			return false
 		case *hclsyntax.ScopeTraversalExpr:
 			return false
 		case *hclsyntax.ConditionalExpr:

